cmd/piper.go

```diff
@@ -93,10 +93,7 @@ func Execute() {
 
 	addRootFlags(rootCmd)
 	if err := rootCmd.Execute(); err != nil {
-		// in case we end up here we know that something in the PreRunE function went wrong
-		// and thus this indicates a configuration issue
```

Comment:
it is no more necessary since due to the update of the generator we set the error category already where the problem happens ...
Thus, no explanation needed here anymore ...